spec/govuk_navigation_helps_spec.rb

```diff
@@ -0,0 +1,84 @@
+require 'spec_helper'
+require 'govuk_navigation_helpers'
+
+RSpec.describe GovukNavigationHelpers::NavigationHelper do
+  describe "#breadcrumbs" do
```

Comment:
Most of the helper method below aren't used twice so perhaps inlining could improve the readability. <oving `described_class.new(content_item).breadcrumbs` into a helper would help too.

Something like this would make the tests understandable without reading anything outside the `it` block:  

```ruby
it "returns the root when parent is not specified" do
   content_item_with_no_parents = {
    "links" => {}
   }

  breadcrumbs = breadcrumbs_for(content_item_with_no_parents)

  expect(breadcrumbs).to eq(
     [
       { title: "Home", url: "/" },
     ]
   )
end
```